public/whiteboard/js/Brush.js

```diff
@@ -0,0 +1,24 @@
+class Brush {
+  constructor (img) {
```

Comment:
assuming u've restructured `main.js` to conform to that alt p5 structure, we're gonna want to add an additional argument to this class's constructor so that we can pass in the reference to p5:
```js
constructor (img, p5) {
  // list brush properties here
  this.img = img
  this.size = 50
  this.color = '#000'
  this.p5 = p5
  this.p5.tint(this.color)
}
```